ec2-spot-elastic-inference/EI_launch_template.json

```diff
@@ -16,7 +16,7 @@
       }
     ],
     "SecurityGroupIds": [
-      "%instanceSecurityGroup%"
+      "sg-0fec95f27280a458c"
```

Comment:
This is specific to your account, we should keep the placeholders. Please don't change those